ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml

```diff
@@ -143,6 +143,13 @@ MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
 BLOB_SIDECAR_SUBNET_COUNT: 6
 
+# Electra
+MAX_BLOBS_PER_BLOCK_ELECTRA: 8
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
+BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: 8
+MAX_REQUEST_BLOB_SIDECARS_ELECTRA: 1024
+TARGET_BLOBS_PER_BLOCK_ELECTRA: 4
```

Comment:
I know the order of this comes from the issue, but it would make more sense to group the two `BLOBS_PER_BLOCK_ELECTRA` variables together.

So put target under max. This applies to all of the yaml files.

Reply:
Yep it was a first rapid copy paste 😄 
Everything should be in the good order now 🤞